atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/KeyValueService.java

```diff
@@ -535,7 +535,7 @@ void createTable(@QueryParam("tableRef") TableReference tableRef, byte[] tableMe
     @Path("put-metadata-for-tables")
     @Consumes(MediaType.APPLICATION_JSON)
     @Idempotent
-    void putMetadataForTables(final Map<TableReference, byte[]> tableRefToMetadata);
+    void putMetadataForTables(Map<TableReference, byte[]> tableRefToMetadata);
```

Comment:
Assuming this was a checkstyle thing, and that you aren't actually modifying the parameter anywhere.